pkg/scheduler/internal/queue/scheduling_queue.go

```diff
@@ -28,6 +28,8 @@ package queue
 
 import (
 	"fmt"
+	utilfeature "k8s.io/apiserver/pkg/util/feature"
```

Comment:
Please also run ./hack/update-gofmt.sh 
This patch will fail CI

Reply:
gofmt didn't complain about this. But please move these two k8s.io imports below "k8s.io/utils/clock" and then run update-gofmt to put it in the right place.